content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -238,4 +238,23 @@ EOF
 )"
 
 ```
-{{% /capture %}}
\ No newline at end of file
+
+## `coredns` pods have `CrashLoopBackOff` or `Error` state
+
+If you have nodes that are running SELinux with an older version of Docker you might experience a scenario
+where the `coredns` pods are not starting. To solve that you can try one of the following options:
+
+- Upgrade to a [newer version of Docker](/docs/setup/independent/install-kubeadm/#installing-docker).
+- Disable SELinux.
+- Modify the `coredns` deployment to set `allowPrivilegeEscalation` to `true`:
+
+```bash
+kubectl -n kube-system get deployment coredns -o yaml | \
+  sed 's/allowPrivilegeEscalation: false/allowPrivilegeEscalation: true/g' | \
+  kubectl apply -f -
+```
+
+Please mind that disabling SELinux or setting `allowPrivilegeEscalation` to `true`
```

Comment:
Let's get rid of "please" (not relevant here, way overused generally throughout these docs) and make the whole thing stronger:

{{< warning >}}
**Warning:**  Disabling SELinux or setting `allowPrivilegeEscalation` to `true` can compromise the security of your cluster.
{{< /warning >}}